lombok.config

```diff
@@ -1,3 +1,4 @@
 # This file is generated by the 'io.freefair.lombok' Gradle plugin
 config.stopBubbling = true
 lombok.addLombokGeneratedAnnotation = true
+lombok.anyConstructor.addConstructorProperties = true
```

Comment:
The property is necessary to make OrganisationPolicy deserialize., but I'm not sure this is needed right here, as I think it gets autogenerated by the configurations I've added in `build.gradle`. So maybe it needs to be added to only one of the files between `lombok.config` and `build.gradle`. If anyone can confirm, I'll delete it from here.